src/System.Memory/tests/MemoryMarshal/TryGetMemoryManager.cs

```diff
@@ -79,7 +79,7 @@ internal class OtherMemoryForTest<T> : MemoryManager<T>
         {
             public OtherMemoryForTest() { }
 
-            public override int Length => 0;
+            public override Memory<byte> Memory => CreateMemory(0);
```

Comment:
It isn't necessary to override this method. Remove it (especially since GetSpan throws NotImplementedException).

Reply:
Done